included/10.3.1/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# WARNING: this file was autogenerated by generate-included-image.js
+# using
+#   npm run add:included -- 10.3.1 cypress/browsers:node14.19.0-chrome100-ff99-edge
+#
+# build this image with command
+#   docker build -t cypress/included:10.3.1 .
+#
+FROM cypress/browsers:node14.19.0-chrome100-ff99-edge
```

Comment:
I think this should be `cypress/browsers:node16.14.2-slim-chrome100-ff99-edge`, that's what `10.3.0` came out using. I don't think this `FROM` has an `arm64` build either.